src/app/components/pages/TeacherOrTutorRequest.tsx

```diff
@@ -38,10 +38,10 @@ export const TeacherOrTutorRequest = ({ user }: { user: RegisteredUserDTO }) =>
                   or a tutor:
                 </p>
                 <ul>
-                  <li>A "teacher" must be directly associated with a school</li>
+                  <li>A &quot;teacher&quot; must be directly associated with a school</li>
```

Comment:
This is much less readable, do we have to replace quotes with &quot; ?

Reply:
I think it's a good rule, here's an explanation of why this is a thing: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md